data/json/items/generic.json

```diff
@@ -107,6 +107,226 @@
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "memory_module",
+    "name": "memory banks",
+    "description": "Allows for storage and recovery of information.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "sensor_module",
+    "name": "sensor array",
+    "description": "A wide range of sensors meant to give the ability to perceive the surrounding world.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "self_monitoring_module",
+    "name": "self monitoring sensors",
+    "description": "A array of sensors and diagnostic modules allowing the robot to perceive itslef.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "central_module",
+    "name": "central integrating module",
+    "description": "This module is responsible for decision making, it's basically the seige of the AI of the robot.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "gun_module",
+    "name": "gun operating system",
+    "description": "This system can operate most conventional weapons.",
+    "copy-from": "robot_module_abstract"
+  },
+  {
+    "type": "GENERIC",
+    "id": "spidery_legs_big",
+    "name": "set of spidery legs",
+    "name_plural": "sets of spidery legs",
+    "description": "A set of big pointy legs, like the ones found under a tripod.",
+    "weight": 40000,
+    "volume": 80,
+    "price": 200000,
+    "material": "steel",
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "spidery_legs_smal",
+    "name": "set of tiny spidery legs",
+    "name_plural": "sets of tiny spidery legs",
+    "description": "A set of tiny pointy legs, like the ones found under a skitterbot.",
+    "weight": 200,
+    "volume": 2,
+    "price": 50000,
+    "material": "steel",
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "reverse_jointed_legs",
+    "name": "set of reverse-jointed legs",
+    "name_plural": "sets of reverse-jointed legs",
+    "description": "A set of reverse-jointed legs, like the ones found under a chicken walker.",
+    "weight": 60000,
+    "volume": 80,
+    "price": 500000,
+    "material": "steel",
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "omni_wheel",
+    "name": "set of omni wheels",
+    "name_plural": "sets of omni wheels",
+    "description": "A set of omni wheels, like the ones found under a police bot.",
+    "weight": 10000,
+    "volume": 40,
+    "price": 100000,
+    "material": "steel",
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "quad_rotors",
+    "name": "set of rotors",
+    "name_plural": "sets of rotors",
+    "description": "A set of rotors able to lift a small drone.",
+    "weight": 100,
```

Comment:
Just 100 grams?

Reply:
what would be more reasonable ?

Reply:
I did some cursory research and he's not all that far off. Considering that CDDA takes place in the future it's not unreasonable to assume that there have been some improvements to motor tech.

Reply:
It's not in the future, though, it's an alternate present. Still, "slightly better than what we have" is what the secret labs crank out.

Reply:
Right, but when I say it's "not far off" I mean he's within 20-30g of the real weight of 4 drone motors, so definitely reasonable.